api/SourceBuffer.json

```diff
@@ -471,10 +471,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "8"
+              "version_added": "11"
```

Comment:
I've tested https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8419 on Safari (macOS) in BrowserStack and see support at least back to 10.1. Before that point I'm not sure because the interface object was not exposed at one point. But at any case the methodology here seems unreliable, so another look would be good.